internal/services/netapp/netapp_volume_resource.go

```diff
@@ -317,6 +318,36 @@ func resourceNetAppVolume() *pluginsdk.Resource {
 				},
 			},
 
+			"data_protection_backup_policy": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"backup_policy_id": {
+							Type:         pluginsdk.TypeString,
+							Required:     true,
+							ValidateFunc: azure.ValidateResourceID,
+							Description:  "The ID of the backup policy to associate with this volume.",
+						},
+
+						"policy_enforced": {
```

Comment:
What're your thoughts on calling this `policy_enabled`? Just to keep it inline with other variables names that are either on or off.

Reply:
Good call, I have changed the public facing property name.